src/store.js

```diff
@@ -4,18 +4,19 @@ import thunkMiddleware from 'redux-thunk';
 
 import rootReducer from './reducers';
 import TRANSIT_API from './utils/Api';
+import HELPERS from './utils/helpers';
 
 /* eslint-disable no-underscore-dangle */
 
 const store = createStore(
   rootReducer,
-  applyMiddleware(thunkMiddleware.withExtraArgument({ TRANSIT_API })),
+  applyMiddleware(thunkMiddleware.withExtraArgument({ TRANSIT_API, HELPERS })),
```

Comment:
I like your approach. One thing to keep in mind though is that HELPERS is currently not being imported on the dashboard-spa side and isn't provided to thunks there. So this will fail in dashboard-spa. Please publish this as an npm module and use it in the dashboard-spa. then create a PR for dashboard-spa alongside this PR. what I would change going forward is: each widget should export an object with key, value pairs that should be put into extra argument in dashboard-spa.

Reply:
@HamidAghdaee In that case, since I don't believe any of the other apps are doing something like this, I almost think it would be better (or at least less obtrusive) to export the functions like you said, and then `import * as HELPERS from '../utils/helpers';` in the actions

Reply:
This as a hacky immediate solution is fine. Please create another trello task to do it right (the general solution I mentioned ) for a later time. thanks.